test/sphere_hyperdiffusion_vec.jl

```diff
@@ -0,0 +1,121 @@
+using Test
+using StaticArrays, IntervalSets
+import ClimaCore.DataLayouts: IJFH
+import ClimaCore:
+    Fields, Domains, Meshes, Topologies, Spaces, Operators, Geometry
+using StaticArrays, IntervalSets, LinearAlgebra
+
+using SphericalHarmonics
+include("sphere_Plm.jl")
+
+FT = Float64
```

Comment:
same here

Reply:
FT and all the operators definitions are also global values, you can move the operators into the function ∇⁴(u), FT should be defined in the testset